src/components/title-block/TitleBlock.styles.ts

```diff
@@ -4,10 +4,12 @@ export const styles = {
     backgroundColor: 'companyBlue',
     borderRadius: 2,
     px: { md: 7, sm: 3, xs: 3 },
-    py: { md: 6, sm: 4, xs: 4 }
+    py: { sm: '48px', xs: 4 },
+    pt: { md: '79px' },
+    pb: { md: '84px' }
   },
   info: {
-    flexBasis: { lg: '55%' }
+    flexBasis: { lg: '58%' }
```

Comment:
I replaced it with minWidth, but I have to leave it because on mockup container with text and button takes more space than image container
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/109922101/dc3bac13-5f71-4b44-8634-0797e5aca3c9)